### DISCUSSION TITLE
Image Grid

### DISCUSSION THREAD

**Discussion by bsdesigner** [lllyasviel/Fooocus#2799] [General]
Good evening! When I set the number of images to 6 and enable the "Generate Image Grid for Each Batch" checkbox, the result is this image with 3 columns and 2 rows:

![x6_1](https://github.com/lllyasviel/Fooocus/assets/36730003/5fcf1ce4-4aa3-4145-8948-32840592f2f8)

What can be changed in the configuration to result in a 2 columns and 3 row image:

![x6_2](https://github.com/lllyasviel/Fooocus/assets/36730003/ffb1046d-6d02-40e8-9909-f37474b39c71)

And actually, is there any way to adjust the number of columns and rows in the image grid?

**Comment by mashb1t** [comment]
@bsdesigner there is no config for the image grid structure, but you can adjust the code in https://github.com/lllyasviel/Fooocus/blob/main/modules/async_worker.py#L110 to modify the amount of columns/rows in general.

**Reply by bsdesigner** [reply]
Thank you very much! It all worked out!!!
